### PR TITLE
feat: add need-finding study with users of geospatial data

### PR DESCRIPTION
**Pull request by parkerziegler**
This PR adds information on our need-finding study with users of geospatial data ito the projects grid! In addition, it includes:

- Documentation pulled from @justinlubin's Slack message on how to run the site locally, so that folks can find this information directly in the repo rather than digging through Slack.
- Some CSS fixes for mobile! Details are below 👇 

## CSS Fixes for Mobile

There were three targeted changes I made to the CSS here. If folks are open to it, I'd also love to add Prettier in a future PR to take care of repo-wide formatting for us (but I also know this can be Controversial ™️ — hence, for another PR).

### Site Hero

The main PLAIT Lab hero / title overflowed mobile screens due to its `font-size`. We now have a `@media` query in place to increase the `font-size` on tablet and above screens.

| Before | After |
| ------ | ------ |
| PLAIT Lab Title – Before | PLAIT Lab Title – After |

### People Grid

The people grid was using hardcoded percentages for `grid-template-columns` which, in combination with a 20px `grid-gap`, meant that this grid would overflow on smaller mobile screens. The full box model math is:

24% grid column width * 4 grid squares = **96%** of nearest relatively positioned parent, or the root of the document
20px grid gap * 3 gaps = **60px**

Fortunately, CSS Grid has flexible, fractional units that represent the amount of available space taking `grid-gap` into account. We can combine this with `repeat` to get definitions like `grid-template-columns: repeat(4, 1fr);` — a grid with four equally sized columns.

The people grid now renders two grid columns on mobile, three on tablet, and four on desktop.

| Before | After |
| ------ | ------ |
|  PLAIT People – Before |  PLAIT People – After |

#### Tablet

PLAIT People – Tablet

#### Desktop

PLAIT People – Desktop

### Projects Grid

This was an identical modification to the one above! Projects are now single column on mobile to really let each one shine (and prevent horizontal overflow), but break back into two column on tablet and above.

| Before | After |
| ------ | ------ |
| PLAIT Projects – Before | PLAIT Projects – After |

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -37,9 +37,6 @@
 
   <script src="https://cdn.jsdelivr.net/npm/vega-tooltip@0.5.1"></script>
   <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/vega-tooltip@0.5.1/build/vega-tooltip.min.css">
-
-  <script src="/assets/js/tag-filtering.js"></script>
```
Comment (parkerziegler) [inline _includes/head.html:41]: I was getting a console warning indicating this file couldn't be found and, indeed, I found no local file pertaining to this. Likewise, we get a warning about this JS file missing in production, so figured we ought to remove this for now!

File: _config.yml [hunk 2]
```diff
@@ -113,12 +113,12 @@ projects:
     desc: Programming-by-demonstration tool for automating repetitive interactions with webpages.  Designed for non-programmers from the social sciences.
     img: /assets/images/projects/helena.png
     website: http://helena-lang.org/
-    displayorder: 4
+    displayorder: 5
```
Comment (parkerziegler) [inline _config.yml:116]: Hope the change in ordering I made is ok — let me know if you want me to just push this project to the end though!

File: _config.yml [hunk 3]
```diff
@@ -27,7 +27,7 @@ descriptionlong: >- # this means to ignore newlines until "baseurl:"
 baseurl: "" # the subpath of your site, e.g. /blog
 url: "https://plait-lab.org" # the base hostname & protocol for your site, e.g. http://example.com
 twitter_username: PLAITlab
-github_username:  plait-lab
+github_username: plait-lab
```
Comment (parkerziegler) [inline _config.yml:30]: These are just little whitespace corrections introduced by my editor due to `formatOnSave` being set to true. Happy to back them out, but looks like it's just removing erroneous whitespace 😄

File: assets/css/style.css [hunk 4]
```diff
@@ -776,10 +783,22 @@ ul.share-buttons .sr-only{
 
 .people{
 	display: grid;
-	grid-template-columns: 24% 24% 24% 24%;
+	grid-template-columns: repeat(2, 1fr);
 	grid-gap: 20px;
 }
 
+@media (min-width: 768px) {
```
Comment (parkerziegler) [inline assets/css/style.css:789]: These are fairly classic tablet and desktop breakpoints, but happy to use a different set if folks prefer. I didn't think we needed to introduce quite as many as something like [Tailwind](https://tailwindcss.com/docs/responsive-design#overview), so I opted to use the two most common ✅

**Comment by justinlubin** [comment]
Awesome, thanks for all the improvements! (And now I'm convinced to try CSS Grid instead of just Flexbox in the future.) As they say in the biz, LGTM!

**Pull request merged**